### ISSUE TITLE
你好？请问可以告诉您代码里面关于CNN卷积的细节处理？

### ISSUE THREAD

**Issue by nutshell999** [KnightZhang625/Enhanced-RCNN_TensorFlow#5]
如果可以的话，我想知道CNN的输入规格以及CNN卷积时卷积核是在哪个维度进行移动？麻烦您了

**Comment by nutshell999** [comment]
`


 W = tf.Variable(tf.truncated_normal(filter_shape, stddev=0.1), name='W')    b = tf.Variable(tf.truncated_normal([1], stddev=0.1), name='b')    conv = tf.nn.conv2d(inputs,                        W,                        strides=[1, 1, 1, 1],                        padding='VALID',                        name='conv')    h = tf.nn.relu(tf.nn.bias_add(conv, b), name='relu')

`

**Comment by nutshell999** [comment]
貌似卷积核不是一个整数型的卷积核，是一个随机正态截取的浮点型卷积核吗？
